docs/datascience/pytorch-support.md

```diff
@@ -1,8 +1,8 @@
 ---
-Order: 9
-Area: python
+Order: 5
+Area: datascience
 TOCTitle: PyTorch Support
-ContentId: 7B5266AD-3D3E-491F-BD7C-B883C592D943
+ContentId: 265ac51a-200d-4016-96da-fa52d8e7ef35
```

Comment:
You can leave the ContentId the same since we're just moving the topic.

Reply:
will do!